tests/src/runtimeApi/module/hipExtModuleLaunchKernel.cpp

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright (c) 2015-2019 Advanced Micro Devices, Inc. All rights reserved.
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANNTY OF ANY KIND, EXPRESS OR
+IMPLIED, INNCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANNY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER INN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR INN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+/* HIT_START
+ * BUILD_CMD: matmul.code %hc --genco %S/matmul.cpp -o matmul.code
+ * BUILD: %t %s ../../test_common.cpp NVCC_OPTIONS -std=c++11
+ * TEST: %t
+ * HIT_END
+ */
+
+
+
+
+#include<chrono>
+#include "hip/hip_runtime.h"
+#include "hip/hip_hcc.h"
+#include "test_common.h"
+using namespace std::chrono;
+
+#define fileName "matmul.code"
+#define kernel_name1 "matmulK"
+#define kernel_name2 "WaitKernel"
+
+//hipDeviceReset() call is used to clear all the allocations in case of any api failure.
+#define HIP_CHECK(status)     \
```

Comment:
Use HIPCHECK from https://github.com/ROCm-Developer-Tools/HIP/blob/master/tests/src/test_common.h#L74

Reply:
Adhered to the rules given in the link and uploaded the changes.